### PR TITLE
fix(variables): define missing css vars

### PR DESCRIPTION
**Pull request by seanforyou23**
This PR adds some missing variable definitions. I'm unsure the values I've used are 100% what they should be, but at least they are set now. Please let me know if there are better values to use.

### CODE REVIEW

File: src/patternfly/components/List/list.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
   --pf-c-content--ul--MarginTop: var(--pf-global--spacer--md);
   --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
   --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
```
Comment (srambach) [inline src/patternfly/components/List/list.scss]: This doesn't match our naming scheme. It should be `--pf-global--ListStyle` It's a typo in _variable.scss instead of here.

File: src/patternfly/components/Content/content.scss [hunk 2]
```diff
@@ -77,7 +77,7 @@
   --pf-c-content--ul--MarginLeft: var(--pf-global--spacer--lg);
   --pf-c-content--ul--nested--MarginTop: var(--pf-global--spacer--sm);
   --pf-c-content--ul--nested--MarginLeft: var(--pf-global--spacer--sm);
-  --pf-c-content--ul--ListStyle: var(--pf-global--ListStyle);
```
Comment (srambach) [inline src/patternfly/components/Content/content.scss]: The correct variable format is `--pf-global--ListStyle`. This should be fixed in _variables.scss.

File: src/patternfly/components/Breadcrumb/breadcrumb.scss [hunk 3]
```diff
@@ -4,7 +4,7 @@
   // Breadcrumb Item
   --pf-c-breadcrumb__item--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-breadcrumb__item--FontWeight: var(--pf-global--FontWeight--semi-bold);
-  --pf-c-breadcrumb__item--LineHeight: var(--pf-global--LineHeight--lg);
```
Comment (srambach) [inline src/patternfly/components/Breadcrumb/breadcrumb.scss]: Why the change from lg to md?

Comment (seanforyou23) [reply src/patternfly/components/Breadcrumb/breadcrumb.scss]: Only because the lg version didn't exist. It's no problem to add it though. Just wasn't sure what the value should be. Also, there's this https://github.com/patternfly/patternfly-next/issues/1109

**Comment by seanforyou23** [comment]
For me, `npm run build-patternfly` no longer works after these changes. I'm not sure why, and kinda need some help to get unstuck. Any ideas? Does anyone else see this besides me?

File: src/patternfly/components/List/list.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/List/list.scss]: This is resolved in https://github.com/patternfly/patternfly-next/pull/1048 btw

**Comment by mcoker** [comment]
@seanforyou23 build seems OK to me - https://paste.fedoraproject.org/paste/WvFZ~BCdGIbYhihyOEFG~A

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 4]
```diff
@@ -63,6 +63,7 @@
   --pf-c-about-modal-box__header--PaddingRight: var(--pf-c-about-modal-box--PaddingRight);
   --pf-c-about-modal-box__header--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-about-modal-box__header--PaddingLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__header--PaddingTop: var(--pf-c-about-modal-box--PaddingTop);
```
Comment (mcoker) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss]: I think you should just delete this here and where it's referenced. It's never used, so I don't see why we would need to account for it in a `min-height`

File: src/patternfly/components/AboutModalBox/about-modal-box.scss [hunk 5]
```diff
@@ -74,6 +75,8 @@
   --pf-c-about-modal-box__content--MarginRight: var(--pf-c-about-modal-box--PaddingRight);
   --pf-c-about-modal-box__content--MarginBottom: var(--pf-c-about-modal-box--PaddingBottom);
   --pf-c-about-modal-box__content--MarginLeft: var(--pf-c-about-modal-box--PaddingLeft);
+  --pf-c-about-modal-box__content--PaddingBottom: var(--pf-c-about-modal-box--PaddingBottom);
+  --pf-c-about-modal-box__content--PaddingTop: var(--pf-c-about-modal-box--PaddingTop);
```
Comment (mcoker) [inline src/patternfly/components/AboutModalBox/about-modal-box.scss]: You can delete both of these and in the line that references them, use `--pf-c-about-modal-box__content--MarginTop` and `--pf-c-about-modal-box__content--MarginBottom`

File: src/patternfly/components/Breadcrumb/breadcrumb.scss (hunk 3 above)
Comment (mcoker) [reply src/patternfly/components/Breadcrumb/breadcrumb.scss]: I would go ahead and change this to `var(--pf-global--LineHeight--sm)` since that's what the design specified, and we can change it via #1109 if it should be different.

File: src/patternfly/components/Chip/chip.scss [hunk 6]
```diff
@@ -42,7 +42,7 @@
   display: inline-flex;
   align-items: center;
   padding-left: var(--pf-c-chip--PaddingLeft);
-  color: var(--pf-c-chip--Color);
+  color: var(--pf-c-chip__text--Color);
```
Comment (mcoker) [inline src/patternfly/components/Chip/chip.scss]: Pretty sure you can just remove this since `.pf-c-chip__text` is a required element of chip. @christiemolloy can you confirm?

Comment (christiemolloy) [reply src/patternfly/components/Chip/chip.scss]: Yes `color: var(--pf-c-chip__text--Color);` is already on the `.pf-c-chip__text` so you can remove this line

**Comment by srambach** [comment]
Looks good! Nice cleanup.
